server/routes/veracode.ts

```diff
@@ -0,0 +1,115 @@
+import { type RequestHandler, Router } from 'express'
+import asyncMiddleware from '../middleware/asyncMiddleware'
+import type { Services } from '../services'
+
+export default function routes({ componentNameService, dataFilterService }: Services): Router {
+  const router = Router()
+
+  const get = (path: string, handler: RequestHandler) => router.get(path, asyncMiddleware(handler))
+
+  get('/', async (req, res) => {
```

Comment:
You're not doing anything with the submitted values from the dropdowns here which is why they don't do anything. I think I did an include file or at least look at another file that uses them and it's routes files.